web/admin.py

```diff
@@ -213,6 +213,6 @@ class EndpointAdmin(admin.ModelAdmin):
 
 @admin.register(Exposure)
 class ExposureAdmin(admin.ModelAdmin):
-    list_display = ('dataset', 'id', 'endpoint', 'form_id')
-    search_fields = ('id', 'dataset', 'endpoint', 'form_id')
+    list_display = ('dataset', 'id', 'endpoint', 'form_id', 'form_name')
+    search_fields = ('id', 'dataset', 'endpoint', 'form_id', 'form_name')
```

Comment:
We should add a filter based on `endpoint__name` here. 

Is it in another PR?

Reply:
@HesamKorki added it